tests/test_tool.py

```diff
@@ -76,8 +79,40 @@ def test_metadata_from_url_rdfa_only(self, mock_get):
         )
         mock_get.return_value = mock_response
 
-        data = metadata_from_url(self.url, microdata=False, jsonld=False, rdfa=True)
-        self.assertEqual(data, expected)
+        data = metadata_from_url(self.url, syntaxes='rdfa')
```

Comment:
I'm not sure, but it seems to me that it would be better to use and support only one way of passing syntaxes (except for default "all"), as a list of valid syntax names, not as a string that contains them.